src/network/networkd-ipv4acd.c

```diff
@@ -227,6 +227,15 @@ int ipv4acd_configure(Link *link, const Address *address) {
 
         log_link_debug(link, "Configuring IPv4ACD for address %s.", IN4_ADDR_TO_STRING(&address->in_addr.in));
 
+        r = address_new(&a);
+        if (r < 0)
+                return r;
+
+        a->family = AF_INET;
```

Comment:
structured initialization here to ensure all fields are correctly initialized

Reply:
`address_new()` initializes the struct with some default values. And, here additional elements (family and so on) are set. So, the code block should be correct.

Upgrading the green label.